src/linear_solvers/indirect.jl

```diff
@@ -2,51 +2,50 @@ struct IndirectSolver <: LinearSolver end
 
 scsint_t(::Type{IndirectSolver}) = Int
 
-function scs_set_default_settings(::Type{IndirectSolver}, data::ScsData{Int})
+function scs_set_default_settings(
+    ::Type{IndirectSolver},
+    stgs::ScsSettings{Int},
+)
     return ccall(
         (:scs_set_default_settings, indirect),
         Cvoid,
-        (Ptr{Cvoid},),
-        data,
+        (Ref{ScsSettings},),
```

Comment:
```suggestion
        (Ptr{Cvoid},),
```
Please keep the API as written in scs.h. We have convert defined now.

Reply:
that's the corresponding `scs.h` line:
```c
void SCS(set_default_settings)(ScsSettings *stgs);
```
also according to https://docs.julialang.org/en/v1/manual/calling-c-and-fortran-code/#man-bits-types
`Ref{...}` is appropriate, since the julia owns the memory.

Why do we want `Ptr{Cvoid}` here?

Reply:
I guess I should have looked at `scs.h` :) I just assumed it was like the other solvers. This is fine.